specification/communication/data-plane/role/preview/2022-01-01-preview/CommunicationRole.json

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
The other folders under `communication/data-plane/` start with an upper-case. I suggest to name the folder `Roles/` (plural)

Reply:
Changed